### PR TITLE
Adding pyproject.toml, minimal setup.py, removed requirements.txt

### PR DESCRIPTION
**Pull request by bnb32**

### CODE REVIEW

File: phygnn/phygnn.py [hunk 1]
```diff
@@ -576,7 +577,7 @@ def fit(self, x, y, p, n_batch=16, batch_size=None, n_epoch=10,
                 logger.debug('Epoch {} batch {} train loss: {:.2e} for "{}"'
                              .format(epoch, b, b_tr_loss, self.name))
 
-            for b, (x_batch, y_batch, p_batch) in enumerate(v_batch_iter):
+            for _, (x_batch, y_batch, p_batch) in enumerate(v_batch_iter):
```
Comment (grantbuster) [inline phygnn/phygnn.py:580]: I think just remove enumerate, right?

File: tests/test_layers.py [hunk 2]
```diff
@@ -525,7 +526,7 @@ def test_gaussian_pooling_train():
                           learning_rate=1e-3,
                           normalize=(True, True))
     model.layers[0].build(xtrain.shape)
-    assert len(model.layers[0].trainable_weights) == 0
+    assert not any(model.layers[0].trainable_weights)
```
Comment (grantbuster) [inline tests/test_layers.py:529]: nitpick but i often prefer len()==0 because if the inside is [0] then we sometimes get messed up on not(any([0]))

**Review by grantbuster — CHANGES_REQUESTED** [review]
minor

File: tests/test_layers.py (hunk 2 above)
Comment (bnb32) [reply tests/test_layers.py:529]: Yeah forgot about that edge case

**Review by grantbuster — APPROVED** [review]
